doc/api/tty.md

```diff
@@ -82,12 +82,15 @@ or `writeStream.rows` properties have changed. No arguments are passed to the
 listener callback when called.
 
 ```js
+process.stdin.setRawMode(true);
 process.stdout.on('resize', () => {
   console.log('screen size has changed!');
   console.log(`${process.stdout.columns}x${process.stdout.rows}`);
 });
 ```
 
+*Note:* Unrealiable event handler execution on all Windows platforms.
```

Comment:
@gibfahn As per [your comment](https://github.com/nodejs/node/pull/13576#discussion_r121152978) I would suggest to include a note about having to use `setRawMode` on Windows.